### ISSUE TITLE
ERROR: Failed building wheel for aiohttp

### ISSUE THREAD

**Issue by Diosagg16** [kennbroorg/iKy#109]
hello, i have this problem in python 3.10.9
if someone knows how to help me plis help me

the error was when I was installing requierements

```
      error: command '/usr/bin/x86_64-linux-gnu-gcc' failed with exit code 1
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for aiohttp
  Building wheel for yagooglesearch (setup.py) ... done
  Created wheel for yagooglesearch: filename=yagooglesearch-1.6.0-py3-none-any.whl size=21138 sha256=3fc30c792b017159aec7f14b00c7a57129a4efc13f3ee5f241d8217671f53141
  Stored in directory: /tmp/pip-ephem-wheel-cache-xjav6lo5/wheels/b3/22/82/2af9489c629fbf15dd4e2cd6841b087cd281a485a223c41b05
  Building wheel for search-engine-parser (setup.py) ... done
  Created wheel for search-engine-parser: filename=search_engine_parser-0.6.3-py3-none-any.whl size=37890 sha256=3da4e48de4d9400afa4e46d97ac4b69ea9b3f73a8cda2e00fc0e8d7d1dbce27c
  Stored in directory: /tmp/pip-ephem-wheel-cache-xjav6lo5/wheels/0d/36/1c/6d0ef01b8f0b18169415f3da6740680f02ff5a07141421a2f3
Successfully built yagooglesearch search-engine-parser
Failed to build aiohttp
ERROR: Could not build wheels for aiohttp, which is required to install pyproject.toml-based projects
```

**Comment by KennBro** [comment]
The quick fix is downgrade python to 3.9 
I am working (As far as my work allows me) on that
The problem is the version of aiohttp needed for some dependencies and an issue of library compilation

**Comment by KennBro** [comment]
Fix it
